File: Queue/JAVA/src/implementing_queue/PriorityQueueHeap.java
package implementing_queue;

import java.util.PriorityQueue;

public class PriorityQueueHeap {
	public static void main(String[] args) {
		PriorityQueue<Integer> myPQH = new PriorityQueue<Integer>();
		myPQH.add(5);
		myPQH.add(0);
		myPQH.add(10);
		myPQH.add(15);

		System.out.println(myPQH.peek());
		myPQH.remove();
		System.out.println(myPQH.toString());
	}
}

File: Queue/Python/PriorityQueueHeap.py
from queue import PriorityQueue

my_pqh = PriorityQueue()

my_pqh.put((2, 'b'))
my_pqh.put((1, 'a'))
my_pqh.put((4, 'd'))
my_pqh.put((3, 'c'))

print(my_pqh.get())
print(my_pqh.get())
